void tomo::IoLoop::run_loop<true, true, false, true, true>()
--- PRE

+++ POST

@@ -609,7 +609,7 @@

 0b8d [ 3] nopl (%rax)
 0b90 [ 5] mov $0x1,%esi
 0b95 [ 3] mov %rbx,%rdi
-0b98 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0b98 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0b9d [ 5] jmp <+0x2f2>
 0ba2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0bad [ 3] nopl (%rax)
